commit 1c6fa285afe436ca2a1f8abd38a6389353f360b6
Author: Sean McArthur <sean@seanmonstar.com>
Date:   Mon Apr 17 14:08:02 2023 -0400

    fix: pending-accept remotely-reset streams pattern was checking is_local

diff --git a/src/proto/streams/state.rs b/src/proto/streams/state.rs
index b9612addc..76638fc87 100644
--- a/src/proto/streams/state.rs
+++ b/src/proto/streams/state.rs
@@ -362,7 +362,7 @@ impl State {
 
     pub fn is_remote_reset(&self) -> bool {
         match self.inner {
-            Closed(Cause::Error(ref e)) => e.is_local(),
+            Closed(Cause::Error(ref e)) => !e.is_local(),
             _ => false,
         }
     }
